### PR TITLE
Styling/homepage/horinzontal card test

### PR DESCRIPTION
**Pull request by Larissartoricader**

### CODE REVIEW

File: components/NavigationBar.js [hunk 1]
```diff
@@ -61,7 +85,10 @@ export default function NavigationBar() {
         onClick={() => router.push("/bookmarks")}
         active={isActive("/bookmarks")}
       >
-        Bookmarks
+        <MenuIconeTitleBox>
+          <StyledHeartIcone />
```
Comment (F-Kirchhoff) [inline components/NavigationBar.js:89]: typo: Icon

File: components/RecipeCard.js [hunk 2]
```diff
@@ -61,43 +117,38 @@ export default function RecipeCard({
...
-        <Image
+        <StyledImage
           src={recipe.image}
-          style={{ width: "100%" }}
           height={200}
           width={200}
-          objectFit="contain"
           alt={recipe.title}
-        ></Image>
+        ></StyledImage>
```
Comment (F-Kirchhoff) [inline components/RecipeCard.js:143]: always use self closing tags if possible. Please check your codebase and update it accordingly.

File: pages/index.js [hunk 3]
```diff
@@ -5,61 +5,52 @@ import styled from "styled-components";
...
-      </TipOfTheDayWrapper>
       <FilteredRecipes
         recipes={recipes}
         bookmarkedRecipesIDs={bookmarkedRecipesIDs}
         onToggleBookmark={onToggleBookmark}
       />
+      {/* <TipOfTheDayWrapper>
+        <Button onClick={handleNextTip}>Get Another Tip</Button>
+        <TipOfTheDay recipe={currentTip}></TipOfTheDay>
+      </TipOfTheDayWrapper> */}
```
Comment (F-Kirchhoff) [inline pages/index.js]: remove this code as well.

File: components/FilteredRecipes.js [hunk 4]
```diff
@@ -88,52 +163,51 @@ const [userInput, setUserInput] = useState("");
...
 
   return (
-    <>
+    <SearchContainer>
       <SearchBox>
-        <h1>Search Recipes</h1>
         <StyledFilterForm onSubmit={handleSearchSubmit}>
-          <label htmlFor="symptom">Select a symptom</label>
-          <input
+          <label htmlFor="symptom"></label>
```
[the remark this answers is not in the document]
Comment (Larissartoricader) [reply components/FilteredRecipes.js:169]: Done!

File: components/FilteredRecipes.js [hunk 5]
```diff
@@ -2,22 +2,99 @@ import { useState } from "react";
...
+  top: 38px;
+  right: 2px;
+  width: 80%;
+  z-index: 1;
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+  max-height: 200px;
+  overflow-y: auto;
+`;
+
+const StyledTextSuggestion = styled.p`
```
[the remark this answers is not in the document]
Comment (Larissartoricader) [reply components/FilteredRecipes.js:64]: Done

File: components/FilteredRecipes.js [hunk 6]
```diff
@@ -88,52 +163,51 @@ const [userInput, setUserInput] = useState("");
...
-          value={userInput}
+            }}
+            value={userInput}
           />
-           <div>
+          <StyledSuggestionsList>
+            {userInput &&
+              symptomSuggestions.map((suggestion, index) => (
+                <StyledTextSuggestion
+                  key={index}
```
[the remark this answers is not in the document]
Comment (Larissartoricader) [reply components/FilteredRecipes.js:185]: done

File: components/FilteredRecipes.js [hunk 7]
```diff
@@ -2,22 +2,99 @@ import { useState } from "react";
...
-  font-size: 12px;
+const StyledSelectedSuggestion = styled.p`
+  background-color: #f1efe2;
+  font-size: small;
+  padding: 5px;
+  border-radius: 10px;
+  display: block;
+`;
+
+const StyledCross = styled.span`
```
[the remark this answers is not in the document]
Comment (Larissartoricader) [reply components/FilteredRecipes.js:86]: done

File: components/NavigationBar.js [hunk 8]
```diff
@@ -47,7 +68,10 @@ export default function NavigationBar() {
   return (
     <StyledNav>
       <StyledLink onClick={() => router.push("/")} active={isActive("/")}>
-        Home
+        <MenuIconeTitleBox>
+          <StyledHomeIcone />
```
[the remark this answers is not in the document]
Comment (Larissartoricader) [reply components/NavigationBar.js:72]: Done

File: components/RecipeCard.js [hunk 9]
```diff
@@ -61,43 +117,38 @@ export default function RecipeCard({
...
+        <StyledBookmarkIcon>
+          <BookmarkIcon
+            onToggleBookmark={onToggleBookmark}
+            bookmarkedRecipesIDs={bookmarkedRecipesIDs}
+            recipe={recipe}
+          />
+        </StyledBookmarkIcon>
```
[the remark this answers is not in the document]
Comment (Larissartoricader) [reply components/RecipeCard.js]: In THIS Case I needed both, but I got it. I will update the code if I see it again ;D

File: components/FilteredRecipes.js [hunk 10]
```diff
@@ -180,20 +181,20 @@ export default function FilteredRecipes({
           />
           <StyledSuggestionsList>
             {userInput &&
-              symptomSuggestions.map((suggestion, index) => (
+              symptomSuggestions.map((suggestion) => (
                 <StyledTextSuggestion
-                  key={index}
+                  key={uid()}
```
Comment (F-Kirchhoff) [inline components/FilteredRecipes.js:186]: this is unfortunately not better. With every render you call uid again, generating a new id. This means that the key changes every render. Please use a fixed value from your data array, for example "suggestion". If suggestion is not unique, please update the data structure to include predefined ids.

Please keep in mind that you need to update this everywhere you use uid this way.

**Review by F-Kirchhoff — CHANGES_REQUESTED** [review]
Hey Herbies,
this looks better, but there is still one issue. please take a look.

**Review by Roland-Hufnagel — APPROVED** [review]
LGTM

**Review by F-Kirchhoff — APPROVED** [review]

**Pull request merged**